src/Site.js

```diff
@@ -531,7 +529,11 @@ Site.prototype.generate = function (baseUrl) {
       .catch((error) => {
         rejectHandler(reject, error, [this.tempPath, this.outputPath]);
       })
-      .finally(() => logger.profile(GENERATE_SITE_LOGGING_KEY));
+      .finally(() => {
+        const endTime = new Date();
+        const totalBuildTime = endTime - startTime;
+        logger.debug(`Total generation time: ${totalBuildTime}ms`);
```

Comment:
Let's:
1. rename this variable to `totalGenerationTime`.
2. use seconds, to be consistent with the existing `totalBuildTime`.
3. remove the `profile` export in logger.js.

Reply:
Should we downgrade the profile logging in `logger.js#profile()` instead, like so:

```js
logger.profile('test', { level: 'debug' });
```

Otherwise, if we are not planning to do any extensive profiling in the future, there's not really a reason for this portion of the code to stay.

Reply:
It seems necessary to upgrade the version of logger (winston) from 2 to 3 in order to execute that code.  ` logger.profile('test', { level: 'debug' }); `
https://github.com/winstonjs/winston/blob/master/UPGRADE-3.0.md

Reply:
Let's just get rid of this part of the code then.